cinema/models.py

```diff
@@ -1,6 +1,18 @@
+import os
+import uuid
+
 from django.core.exceptions import ValidationError
 from django.db import models
 from django.conf import settings
+from django.utils.text import slugify
+
+
+def instance_image_file_path(instance, filename):
+    _, extension = os.path.splitext(filename)
+
+    filename = f"{slugify(instance.title)}-{uuid.uuid4()}.{extension}"
```

Comment:
you have double dots here, remove one from f-string

Reply:
oh, you didn't remove an extra dot `.`
look at the name of the image when you add it